pkg/flowaggregator/flowaggregator.go

```diff
@@ -360,18 +345,12 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	// Currently, we send 4 templates for covering all the cases in dual-stack clusters, where Pod IPs
-	// and original exporter IP could belong to different IP families.
-	if fa.templateIDv4Expv4, err = fa.createAndSendTemplate(false, false); err != nil {
+	// Currently, we send two templates for IPv4 and IPv6 regardless of the IP families supported by cluster
+	fa.createAndSendTemplate(true)
```

Comment:
why does this need to be called two times? is this an expected line?

Reply:
Thanks for noticing this. This should be removed.